src/components/Drop-Down/HelpMenu.tsx

```diff
@@ -3,17 +3,17 @@ import React, { useRef, useState } from 'react';
 const HelpMenu: React.FC = function () {
```

Comment:
Where does this get used?

Reply:
I think Joe may be waiting on design from me here.. there was a placeholder component that was a dropdown to contain the instructions to use the Playground. but i left it out of the last mockup by accident.